processor/intervalprocessor/config.go

```diff
@@ -37,9 +38,13 @@ type PassThrough struct {
 // Validate checks whether the input configuration has all of the required fields for the processor.
 // An error is returned if there are any invalid inputs.
 func (config *Config) Validate() error {
-	if config.Interval <= 0 {
+	if config.Interval <= time.Second {
 		return ErrInvalidIntervalValue
 	}
 
+	if config.Interval%time.Second != 0 {
```

Comment:
I mean, choosing an interval of like `16s345ms` is *odd* for sure. But I don't think it's necessarily a problem for the algorithm